src/status_im2/contexts/profile/events.cljs

```diff
@@ -27,6 +27,13 @@
             {:db (assoc db :profile/profiles-overview profiles)}
             (profile-selected key-uid)))
 
+(rf/reg-event-fx
+ :profile/switch-profile
+ (fn [{:keys [db]} [key-uid]]
+   {:db db
+    :fx [[:dispatch [:profile/profile-selected key-uid]]
```

Comment:
same here, its better just dispatch in the view no need in the `:profile/switch-profile` event